### PR TITLE
DigitizerWorkflow: Enable Configurable parameters to be influenced

### PR DESCRIPTION
**Pull request by sawenzel**
With this, configurable parameters can be changed from the command line
and used in all processors participating in the digitizer workflow.

This is a first step. We are discussing a yet deeper automation
of the mechanism.

### CODE REVIEW

**Comment by sawenzel** [comment]
@shahor02 , @ktf: As discussed.

**Comment by sawenzel** [comment]
Some new problem slipped into the CI:
```
=== /macro/eventDisplay.C - Process running as 10612 for attempt 1 ===

cling::DynamicLibraryManager::loadLibrary(): libOpenGL.so.0: cannot open shared object file: No such file or directory
Error in <TInterpreter::TCling::AutoLoad>: failure loading library EventDisplay.so for FairEventManager
cling::DynamicLibraryManager::loadLibrary(): libOpenGL.so.0: cannot open shared object file: No such file or directory
Error in <TInterpreter::TCling::AutoLoad>: failure loading library EventDisplay.so for FairEventManager
cling::DynamicLibraryManager::loadLibrary(): libOpenGL.so.0: cannot open shared object file: No such file or directory
Error in <TInterpreter::TCling::AutoLoad>: failure loading library EventDisplay.so for FairEventManager
cling::DynamicLibraryManager::loadLibrary(): libOpenGL.so.0: cannot open shared object file: No such file or directory
Error in <TInterpreter::TCling::AutoLoad>: failure loading library EventDisplay.so for FairEventManager
cling::DynamicLibraryManager::loadLibrary(): libOpenGL.so.0: cannot open shared object file: No such file or directory
Error in <TInterpreter::TCling::AutoLoad>: failure loading library EventDisplay.so for FairEventManager
In file included from input_line_10:1:
```
@dberzano : any idea?

File: Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx [hunk 1]
```diff
@@ -280,6 +286,13 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   // workflow in the upper left corner of the GUI.
   WorkflowSpec specs(1);
 
+  // Update the (declared) parameters if changed from the command line
+  // Note: In the future this should be done only on a dedicated processor managing
+  // the parameters and then propagated automatically to all devices
```
Comment (shahor02) [inline Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx]: Do you mean to keep these line here only to write the ``ini`` file and to add to all devices which may need configuration:
in ``DataProcessorSpec getXXXSpec()``:
````
Options{
...
{ "configKeyValues", VariantType::String, "", { "KeyValue params" } }
}
````
and in the ``init()``:
``o2::conf::ConfigurableParam::updateFromString( ic.options().get<std::string>("configKeyValues")) ;`` ?

Comment (sawenzel) [reply Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx]: Something along those lines...

**Pull request merged**